__tests__/adapter.test.ts

```diff
@@ -65,15 +66,22 @@ vi.mock('node-fetch', () => ({
     Response
 }));
 
-describe("StorachaAdapter", () => {
-    let adapter: StorachaAdapter;
+async function createTestAgent() {
+    const principal = await Signer.generate();
+    return {
+        agentPrivateKey: principal.signer.verifier,
+        delegation: await principal.delegation()
```

Comment:
We spoke about maybe changing this to `agentDelegation` for consistency?

Reply:
Indeed. Missed that one. Thanks for flagging.